### PR TITLE
Translation of default README to PT-BR

### PR DESCRIPTION
**Pull request by ghost**
If you need, I can translate to other languages too.

### CODE REVIEW

File: README-PT-BR.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Visual Studio Code - Open Source
+
+[![Build Status](https://vscode.visualstudio.com/_apis/public/build/definitions/a4cdce18-a05c-4bb8-9476-5d07e63bfd76/1/badge?branchName=master)](https://aka.ms/vscode-builds)
+[![Feature Requests](https://img.shields.io/github/issues/Microsoft/vscode/feature-request.svg)](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc)
+[![Bugs](https://img.shields.io/github/issues/Microsoft/vscode/bug.svg)](https://github.com/Microsoft/vscode/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug)
+[![Gitter](https://img.shields.io/badge/chat-on%20gitter-yellow.svg)](https://gitter.im/Microsoft/vscode)
+
+[VS Code](https://code.visualstudio.com) é um novo tipo de ferramenta que combina a simplicidade de
+um editor de código com o que desenvolvedores precisam para o seu ciclo principal de edição-compilação-deburação. VS Code
```
Comment (coreh) [inline README-PT-BR.md]: Where it says `deburação`, it should be `depuração`, like on the line below.

File: README-PT-BR.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+* Faça uma pergunta no [Stack Overflow](https://stackoverflow.com/questions/tagged/vscode).
+* Peça uma nova função no [GitHub](CONTRIBUTING.md).
+* Vote para [pedidos populares de novas funções](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
+* Notifique sobre bugs em [GitHub Issues](https://github.com/Microsoft/vscode/issues).
+* [Tweet](https://twitter.com/code) nos com outros comentários.
+
+## Projetos Relacionados
+
+Muitos dos principais componentes e extensões do Code estão em seus próprios repositórios no GitHub. Por exemplo, o [adaptador de depuração do nó](https://github.com/microsoft/vscode-node-debug) e o [adaptador de depuração mono](https://github.com/microsoft/vscode-mono-debug).
```
Comment (coreh) [inline README-PT-BR.md]: "Nó" -> Node?

File: README-PT-BR.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+[![Bugs](https://img.shields.io/github/issues/Microsoft/vscode/bug.svg)](https://github.com/Microsoft/vscode/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug)
+[![Gitter](https://img.shields.io/badge/chat-on%20gitter-yellow.svg)](https://gitter.im/Microsoft/vscode)
+
+[VS Code](https://code.visualstudio.com) é um novo tipo de ferramenta que combina a simplicidade de
+um editor de código com o que desenvolvedores precisam para o seu ciclo principal de edição-compilação-deburação. VS Code
+fornece suporte abrangente de edição e depuração, um modelo de extensibilidade e integração leve com ferramentas existentes.
+
+VS Code é atualizado mensalmente com novas funções e correções de bugs. Você pode baixá-lo para Windows, macOS e Linux no site oficial do[VS Code](https://code.visualstudio.com/Download). Para ter os últimos lançamentos todos os dias, você pode instalar a [Versão Insiders do VS Code](https://code.visualstudio.com/insiders). Essa compilação é feita a partir da branch master e é atualizada diariamente.
```
Comment (coreh) [inline README-PT-BR.md]: Missing space before the [VS Code] link

File: README-PT-BR.md (hunk 2 above)
Comment (gugabrgx) [reply README-PT-BR.md]: It is the portuguese equivalent to NODE

File: README-PT-BR.md (hunk 3 above)
Comment (gugabrgx) [reply README-PT-BR.md]: Solved too, sorry for this inconvenience.

**Review by ghost — COMMENTED** [review]
All the bugs that you appointed are now solved.

**Comment by egamma** [comment]
@gugabrgx thanks for the translations, but we have currently no plans to translate our development documentation on Github.

**Pull request closed without merging**